Source/Scene/Expression.js

```diff
@@ -104,7 +104,16 @@ define([
         asin : Math.asin,
         atan : Math.atan,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        degrees : CesiumMath.toDegrees,
+        sign : Math.sign,
+        floor : Math.floor,
+        ceil : Math.ceil,
+        round : Math.round,
+        exp : Math.exp,
+        exp2 : exp2,
+        log : Math.log,
+        log2 : Math.log2,
```

Comment:
It turns out this is also not supported for IE. `log2` will also need to be a local function which can use `CesiumMath.logBase`.